lab1.js

```diff
@@ -54,29 +54,45 @@ assert(1 === 2, 'this is an assertion failure example. 1 === 2');
  it failed.
 */
 
-//your code goes here
+assert("tigers" > "elephants");
+assert("elephants" > "penguins", "Penguins are obviously cooler than elephants.");
 
 /* ----------------- Meerkats -------------------------------------------------
  Meerkats make a sort of chirping noise (according to my 30 seconds of
  research).  We're going to translate two sentences into meerkat speech.
 */
 
-var sentence1 = 'More food please.';
-var sentence2 = 'Come over here so you can scratch my belly.';
+var sentence1 = "More food please.";
+var sentence2 = "Come over here so you can scratch my belly.";
 
 /*
  TODO: 20 points
- Your goal is to replace the words in the above sentences with 'chirp' The
+ Your goal is to replace the words in the above sentences with "chirp" The
  assertions at the end of this section should pass when you're done.  Use
  **two** different kinds of loops to implement this. (10 points each.)
- HINT: the 'split' method on String will be useful.
+ HINT: the "split" method on String will be useful.
 */
 
-//your code goes here
+  var w1 = sentence1.split(" ");
+  for (var i = 0; i < w1.length; i++) {
+    w1.splice(0, 1);
+    w1.push("chirp");
```

Comment:
I like this - not at all the logic I would have come up with...and it totally works. Nice.